src/js/index.js

```diff
@@ -1423,15 +1394,45 @@ function accountIconShowHide(){
 
 function setPrivacy() {
   var checkBox = document.getElementById("privacy-policy-checkbox");
+  var checkBoxRegister = document.getElementById("privacy-policy-form-checkbox-register");
+  var checkBoxLogin = document.getElementById("privacy-policy-form-checkbox-login");
 
-  if (checkBox.checked == true){
+
+  if (checkBox.checked == true || checkBoxRegister.checked == true || checkBoxLogin.checked == true){
```

Comment:
There is still an issue where if I have accidentally checked the box in the register screen, then:
1. switch to the login screen
2. untick the box
3. login
4. click on the bottom right to open the user menu
5. checkbox is checked, when it should be unchecked!

We could either have a function that is called when one box is checked/unchecked, that checks/unchecks all of them automatically or have a single internal state that these checkboxes always reflect. Or something else that I haven't thought of 😄

Reply:
Hey Thano, thanks a lot for the comment. Good job for pointing that out. I think i have fixed it now, so, could you please check again?

Reply:
Works great now. Approving the PR...